updater.py

```diff
@@ -48,6 +52,7 @@ def deus_updator(managers):
         else:
             print('TOTAL SUPPLY:   ', total_supply)
             print('NON-CIRCULATING:', nc_supply)
+            print('CIRCULATING:', total_supply- nc_supply)
```

Comment:
According to PEP 8 guidelines, it is recommended to include a single space on both sides of binary operators.

Reply:
Addressed the issue accordingly